src/components/Calculator.js

```diff
@@ -1,76 +1,188 @@
 import React from 'react';
 import './Calculator.css';
+import { calculate, isNumber } from '../logic/calculate';
 
 class Calculator extends React.Component {
   constructor(props) {
     super(props);
-    this.state = '';
+    this.state = { toDisplay: 0 };
+    this.obj = {
+      total: null,
+      next: null,
+      operation: null,
+    };
   }
 
+  handleClick = (event) => {
+    const buttonNm = event.target.name;
+    this.obj = calculate(this.obj, buttonNm);
+    if (isNumber(buttonNm) || buttonNm === '+/-') {
+      this.setState({ toDisplay: this.obj.next || 0 });
+    } else {
+      this.setState({ toDisplay: this.obj.total || 0 });
+    }
+  };
+
   render() {
+    const { toDisplay } = this.state;
     return (
       <section className="calculator-container flex-row">
         <div className="output-container flex-row">
-          <p className="output">0</p>
+          <p className="output">{toDisplay}</p>
         </div>
         <div className="left-buttons flex-row">
-          <button type="button" className="left-button">
+          <button
+            name="AC"
+            type="button"
+            className="left-button"
+            onClick={this.handleClick}
+          >
             AC
           </button>
-          <button type="button" className="left-button">
+          <button
+            name="+/-"
+            type="button"
+            className="left-button"
```

Comment:
[OPTIONAL] I think your code will be neater and less error-prone if you used an object coupled with a button component. Instead of hardcoding the numbers.